service/src/main/java/bio/terra/landingzone/service/landingzone/azure/model/LandingZoneRecord.java

```diff
@@ -0,0 +1,58 @@
+package bio.terra.landingzone.service.landingzone.azure.model;
```

Comment:
LandingZoneRecord sounds like something from database. But we already have LandingZone model for DB. Probably it doesn't make sense to swap them. Hope it won't confuse us in future.